mod/NomaiTextQoL.cs

```diff
@@ -140,7 +135,7 @@ public static bool NomaiTextLine_DetermineTextLineColor_Prefix(NomaiText __insta
         public static bool NomaiWallText_SetAsTranslated_Prefix(NomaiWallText __instance, ref int id)
         {
             if (!AutoNomaiText) return true;
-            if (ManualScrollLocations.ContainsKey(__instance.gameObject.name)) return true;
+            if (LocationTriggers.ManualScrollLocations.ContainsKey("Arc_" + __instance.gameObject.name)) return true;
```

Comment:
Oh I didn't realize the names here had Arc_. I think we're adding the Arc_s in the wrong place though; doesn't ManualScrollLocations now have "TT_Tower_CT" while this go.name is "Arc_TT_Tower_CT", so we're doing ContainsKey(Arc_Arc_TT_Tower_CT)?

Reply:
I think what we want is `var name = __instance.gameObject.name; if (name.StartsWith("Arc_") && LocationTriggers.ManualScrollLocations.ContainsKey(name.Substring(4))) ...` ?

Reply:
Oh oops you're right

Reply:
Fixed